docs-src/releases.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Releases
+
+## Official Custom Resource Definitions
+
+The custom resources defined as part of Service APIs will be considered official
+custom resource definitions (CRDs). That means that they will go through a KEP
+review process and be part of a Kubernetes project. 
+
+This is a relatively new concept, and there is only one previous example of this
+being done:
+[VolumeSnapshots](https://kubernetes.io/blog/2018/10/09/introducing-volume-snapshot-alpha-for-kubernetes/). These definitions are
+automatically installed by the CSI drivers that support them. This pattern
+doesn't make as much sense here where we want to support multiple controllers
+per cluster.
+
+CRDs do not need to be tied to Kubernetes versions, the release cycle is
+independent. The primary requirement of an official CRD is that it goes through
+the KEP process. Versioning should be well thought out, with several alphas, and
+a great deal of API stability before releasing a beta.
```

Comment:
We probably want to be more specific here. Isn't this basically the Kubernetes API stability contract? Maybe just reference that, but with a faster iteration period than k8s core release cadence?

Reply:
Updated, thanks! What I linked to does not talk about release cadence, just API versioning requirements.